client/src/components/widgets/ContextMenu/script.js

```diff
@@ -60,6 +59,12 @@ export default {
       }
       return [0, 0];
     },
+    offsetPos() {
+      if (this.itemInfo?.isPlotly) {
+        return [this.pos[0] + 135, this.pos[1] + 150];
```

Comment:
magic numbers... Do you want to gather them at the top of the file as `const OFFSET_PLOTLY=[90, 50];`